### PR TITLE
Updated to latest ExpressRoute specs

### PR DESCRIPTION
**Pull request by nupurkot**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nupurkot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nkothar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ExpressRouteManagement/ExpressRouteManagement.csproj [hunk 1]
```diff
@@ -12,6 +12,8 @@
     <OutputType>Library</OutputType>
     <RestorePackages>true</RestorePackages>
     <NuGetPackageImportStamp>56133d78</NuGetPackageImportStamp>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
```
Comment (ogail) [inline src/ExpressRouteManagement/ExpressRouteManagement.csproj]: Remove extra tag

**Comment by ogail** [comment]
Update nuget version

**Comment by nupurkot** [comment]
Addressed both comments. Thanks!

**Pull request merged**